### PR TITLE
Use proper dependency between GitExtensions and TranslationApp

### LINKED ISSUE #2743: Fix intermittent build warning file GitExtension\bin\Debug\Translation\Spanish.xlf is used by another process
The issue is intermittent and each time different translation file is failing

### PR DESCRIPTION
**Pull request by mnaoumov**
Fixes #2743

### CODE REVIEW

**Comment by KindDragon** [comment]
Why GitExt should depend on TranslationApp?

**Comment by mnaoumov** [comment]
@KindDragon only to fix underlying compilation issue #2743. TranslationApp keeps its files in GitExtensions folder and since projects were independent, build of those projects might cause race conditions

File: GitExtensions/GitExtensions.csproj [hunk 1]
```diff
@@ -185,6 +185,10 @@
       <Project>{D3440FD7-AFC5-4351-8741-6CDBF15CE944}</Project>
       <Name>ResourceManager</Name>
     </ProjectReference>
+    <ProjectReference Include="..\TranslationApp\TranslationApp.csproj">
```
Comment (sharwell) [inline GitExtensions/GitExtensions.csproj]: :exclamation: Use solution dependencies, not project references for this.

**Comment by sharwell** [comment]
In Visual Studio, right click the **GitExtensions** project and select **Build Dependencies** &rarr; **Project Dependencies...**. Then check the box for any projects which the **GitExtensions** project depends on.

**Comment by mnaoumov** [comment]
@sharwell , yes, I will implement that. The reason why I did not do this at fiest place is the bug in MSBuild 4.0 which ignores solution-wise project dependencies https://connect.microsoft.com/VisualStudio/feedback/details/586875/msbuild-4-0-incorrectly-processes-project-dependencies-specified-in-solution-file

**Comment by sharwell** [comment]
I believe simply installing .NET 4.5 is enough to correct this. If you can't do that (e.g. running Windows XP?), you can omit the `/m` flag (or disable parallel builds in Visual Studio) to work around the issue.

**Pull request merged**